libs/editor/egui_editor/rustfmt.toml

```diff
@@ -0,0 +1,4 @@
+use_small_heuristics = "Max"
+chain_width = 60
+fn_args_layout = "Compressed"
+use_field_init_shorthand = true
```

Comment:
Also a file I'd say to drop or merge with root level one.

Reply:
yep dropping